web/src/connectors/model-detailed-view-connector/model-detailed-view-connector.tsx

```diff
@@ -21,20 +22,24 @@ export const ModelDetailedViewConnector: React.FC<ModelDetailViewProps> = ({
 
     const { data: model } = useModelById({ modelId, modelVer }, {});
 
-    const { data: training } = useModelTraining({ trainingId: model?.training_id || 1 }, {});
+    // TODO: fix hooks - the hook is currently not working correctly with the server
```

Comment:
Do we need a task for this?
Looks like it can produce bugs or something

Reply:
add issue https://github.com/epam/badgerdoc/issues/571